terms-of-service.md

```diff
@@ -1,3 +1,90 @@
-# Terms
+# Terms of Service
+
+---
+title: Terms
+order: 4
+---
+
+Hey there! We're Bloggify and we're happy that you are here, on our website. When we say **we**, **us** or **Bloggify** it's because that's who we are. We own and run the Bloggify website.
+
+By navigating on this website you agree to follow the the terms explained on this page. Please read and understand them.
+If you don't accept them, then we'll be sorry, but you will need to leave.
+
+You promise that information you give us is true, accurate and complete.
+
+In the Checkout process, we will ask you to provide your real name, billing address, email address and the credit card information. The card information is *not* stored on our side but securely sent to [Braintree](https://braintreepayments.com)– the service which handles the credit card payments.
+
+Your real name and address will be used for invoicing.
+
+### How buying things works
+
+In this this section we will explain how the process works when you decide to buy something on our website.
+
+Some of products sold on Bloggify are owned by us and some are owned by other authors. You will find that information in each product description.
+
+What are you actually buying? When you buy something on our website, you are not acquiring the actual item, instead you acquire the right or a license to use that item. The license is granted to you directly by the author. Make sure you understand the license type for the item you are buying. If you are letting other people use your item, make sure you choose the right license to allow that use.
+
+The products are listed on the website. By visiting the product page you can open add it to the cart. If you want to buy more items, that's possible too.
+
+The product price is listed on the product page.
+
+On the Checkout page you will see the amounts for each item and the grand total. The payment is done via credit card. The payment is processed by [Braintree](https://braintreepayments.com).
+
+The currency is euro (EUR).
+
+When you buy something, you are doing so on the following terms:
+
+ - you promise you considered the suitability of your chosen license, and that you have chosen appropriately
+ - you cannot cancel a completed purchase of an item
+ - we do not promise that any particular item will continue to be available, so you should download and save the item as soon as you buy it
+ - you understand that others might also bought a particular item under the same license
+ - the author retains ownership of the item
+ - your relevant details may be provided to the author of the item in order to facilitate the transaction (generally for invoicing and item support)
+ - the billing information you provide is correct and complete
+
+### Refunds
+
+We do *not* grant refunds on the purchases made on this website.
```

Comment:
@lex124 We can mention here that people may contact us for solving disputes/problems.